src/components/DotaMap/DotaMap.css

```diff
@@ -0,0 +1,23 @@
+.map {
```

Comment:
Nah, let's leave this file as it is. We never know when we want to use both.

But yeah, agree on the other point.